setup.cfg

```diff
@@ -39,6 +39,9 @@ install_requires =
 	scipy
 	pytorch-lightning >= 1.7
 	torch >= 1.11
+	typed-argument-parser
+    tensorboardX
+    hyperopt
```

Comment:
these packages aren't needed in v2? They're only needed currently because we still have the v1 code in there (which will be deleted by the `2.0.0a` release).